config/taxonomy/policymanager.values.schema.json

```diff
@@ -4,7 +4,7 @@
     "title": "Policy Manager Values Taxonomy",
```

Comment:
These are values for the policy manager **request** values. Name of file and title should be changed accordingly

Reply:
Let me correct it.

Reply:
Done